ompi/mca/coll/han/coll_han_allreduce.c

```diff
@@ -132,7 +132,7 @@ mca_coll_han_allreduce_intra(const void *sbuf,
                                 seg_count);
 
     /* Determine number of elements sent per task. */
-    OPAL_OUTPUT_VERBOSE((10, mca_coll_han_component.han_output,
+    OPAL_OUTPUT_VERBOSE((30, mca_coll_han_component.han_output,
```

Comment:
?

Reply:
It seemed rather noisy when I was trying to view my own logging.  It has its own commit, maybe a separate PR would be better.

Reply:
now in https://github.com/open-mpi/ompi/pull/12842